pkg/policies/security/security.go

```diff
@@ -0,0 +1,206 @@
+// Copyright 2021 Allstar Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package security implements the SECURITY.md security policy.
+package security
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"path"
+
+	"github.com/ossf/allstar/pkg/config"
+	"github.com/ossf/allstar/pkg/config/operator"
+	"github.com/ossf/allstar/pkg/policydef"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/rs/zerolog/log"
+)
+
+const configFile = "security.yaml"
+const polName = "SECURITY.md"
```

Comment:
The policy name being `SECURITY.md` is intentional or typo? Maybe something like `Security`?

Reply:
I debated this. GitHub calls having a SECURITY.md file a "Security Policy", so that would make sense to call this policy "Security". However, in Allstar we are calling it a "Security Policy App" are categorizing all the policies as "Security Policies", so I need to differentiate this one.